code/game/machinery/launch_pad.dm

```diff
@@ -145,6 +145,12 @@
 	for(var/atom/movable/ROI in source)
 		if(ROI == src)
 			continue
+		// Dont teleport unanchored ais
+		if(isAI(ROI))
+			var/mob/living/silicon/ai/I = ROI
+			if(I.move_resist > 1000)
+				to_chat(user, "<span class='warning'>ERROR: Launchpad overloaded, unable to operate.</span>")
+				return
```

Comment:
just uh make it do `continue` to skip the AI that way other things on the same tile as the AI will still teleport

Reply:
With current code it still teleports items, here:

https://gyazo.com/fe2ae4c6cd8ad2e7d35aedb4c875c60a

Reply:
Done as requested

Reply:
https://github.com/yogstation13/Yogstation/pull/7960/commits/22105ea49ee7d53af8de64a304625d1322bb9be7